toonz/sources/toonzlib/tstageobjectspline.cpp

```diff
@@ -226,6 +290,15 @@ void TStageObjectSpline::saveData(TOStream &os) {
     os << p.x << p.y << p.thick;
   }
   os.closeChild();
+  if (m_externalDataFound) {  // Needed for T2D compatibility
+    os.openChild("interpolationStroke");
+    n = interpStroke.size();
```

Comment:
I think you can simply use `m_interpolationStroke` directly instead of assigning the values to `interpStroke`.
(i.e. you can remove the function `TStageObjectSpline::getInterpolationStroke()` )